<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomServerConnectionWidget</class>
 <widget class="QDialog" name="CustomServerConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Databases from a Server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="serverConnectionTab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Postgis</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="ExploreServerWidget" name="serverWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="postgisEdgvComboFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Load Database Model EDGV Version 2.1.3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Load Database Model EDGV Version 2.1.3 Pro</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Load Database Model EDGV Version 3.0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Load Database Model EDGV Version 3.0 Pro</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Load Database Model EDGV Version FTer_2a_Ed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Load Other Database Models</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="CustomSelector" name="postgisCustomSelector" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Spatialite</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="CustomSelector" name="spatialiteCustomSelector" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="SelectFileWidget" name="spatialiteCustomFileSelector" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="spatialiteEdgvComboFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Load Database Model EDGV Version 2.1.3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Load Database Model EDGV Version 3.0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Load Database Model EDGV Version FTer_2a_Ed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Load Other Database Models</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectFileWidget</class>
   <extends>QWidget</extends>
   <header>DsgTools.gui.CustomWidgets.SelectionWidgets.selectFileWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExploreServerWidget</class>
   <extends>QWidget</extends>
   <header>DsgTools.gui.CustomWidgets.ConnectionWidgets.ServerConnectionWidgets.exploreServerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomSelector</class>
   <extends>QWidget</extends>
   <header>DsgTools.gui.CustomWidgets.SelectionWidgets.customSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
